v22.1/take-full-and-incremental-backups.md

```diff
@@ -150,36 +154,36 @@ Periodically run the [`BACKUP`][backup] command to take a full backup of your cl
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-> BACKUP INTO '{collection location}';
+> BACKUP INTO '{collectionURI}';
 ~~~
 
-Then, create nightly incremental backups based off of the full backups you've already created. To append an incremental backup to the most recent full backup created in the [collection's](#backup-collections) storage location, use the `LATEST` syntax:
+Then, create nightly incremental backups based off of the full backups you've already created. To append an incremental backup to the most recent full backup created at the [collection's](#backup-collections) URI, use the `LATEST` syntax:
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-> BACKUP INTO LATEST IN '{collection location}';
+> BACKUP INTO LATEST IN '{collectionURI}';
 ~~~
 
-<span class="version-tag">New in v22.1:</span> This will add the incremental backup to the default `/incrementals` directory at the root of the backup collection's directory. With incremental backups in the `/incrementals` directory, you can apply different lifecycle policies from cloud storage providers to the `/incrementals` directory as needed.
+<span class="version-tag">New in v22.1:</span> This will add the incremental backup to the default `/incrementals` directory at the root of the backup collection's directory. With incremental backups in the `/incrementals` directory, you can apply different lifecycle/retention policies from cloud storage providers to the `/incrementals` directory as needed.
 
 {{site.data.alerts.callout_info}}
-If there are existing incremental backups in the storage location taken with v21.2 or earlier, new incremental backups will add to the existing incremental directory, (e.g., `{collection location}/{subdirectory}`). This follows how [incremental backups](take-full-and-incremental-backups.html#take-an-incremental-backup) were stored by default in v21.2 and earlier. To back up using the prior behavior, see this [example](#backup-earlier-behavior) for details on using the `incremental_location` option to achieve this.
+If there are existing incremental backups in the storage location taken with v21.2 or earlier, new incremental backups will add to the existing incremental directory, (e.g., `{collectionURI}/{subdirectory}`). This follows how [incremental backups](take-full-and-incremental-backups.html#take-an-incremental-backup) were stored by default in v21.2 and earlier. To back up using the prior behavior, see this [example](#backup-earlier-behavior) for details on using the `incremental_location` option to achieve this.
```

Comment:
I think this could be a little clearer. Something along the line of: "In v21.2 and earlier, incremental backups were stored in the same directory as their full backup (i.e. collectionURI/subdirectory). If an incremental backup command points to a subdirectory with incremental backups created in v21.2 and earlier, v22.1 will write the incremental backup to the v21.2 default location).